contributors.json

```diff
@@ -110,6 +110,15 @@
       "location": "Aix-en-Provence",
       "hireable": true
     },
+    "dguyon@gmail.com": {
```

Comment:
He made some commits so yes it should be here. We should just had removed his .json file.